dataprofiler/labelers/data_processing.py

```diff
@@ -825,8 +825,8 @@ def _word_level_argmax(self, data, predictions, label_mapping,
             label_count = {label_mapping[default_label]: 0}
             for idx in range(len(sample)):
```

Comment:
Would an `enumerate` not be cleaner?

Reply:
agreed, but not going to block the PR if @lettergram doesn't get to it prior to merge